pkg/engine/update.go

```diff
@@ -139,13 +139,19 @@ type UpdateOptions struct {
 	// Specific resources to replace during an update operation.
 	ReplaceTargets deploy.UrnTargets
 
+	// Specific resources to skip updating during a deployment.
+	Excludes deploy.UrnTargets
+
```

Comment:
Move this below `TargetDependents` maybe?  Otherwise the comment below with the `XXXTargets lists` is becoming more confusing than it already is.